package.json

```diff
@@ -13,15 +13,17 @@
     "routr": "^0.0.x"
   },
   "devDependencies": {
-    "chai": "^1.10.0",
+    "chai": "^2.0.0",
     "coveralls": "^2.11.1",
+    "fluxible": "^0.2.2",
     "fluxible-app": "^0.1.0",
     "grunt": "^0.4.5",
     "istanbul": "^0.3.2",
     "jshint": "^2.5.5",
     "mocha": "^2.0.1",
     "mockery": "^1.4.0",
-    "precommit-hook": "1.0.x"
+    "precommit-hook": "1.0.x",
+    "react": "^0.12.2"
```

Comment:
I think React got added here because it's part of the peer deps of Fluxible? Do we want to remove this?